projectTemplates/ts-server/src/serverBuilder.ts

```diff
@@ -33,6 +34,7 @@ export class ServerBuilder {
   }
 
   private registerMiddleware(): void {
+    this.serverInstance.use(cors());
```

Comment:
Move `body-parser` under `cors` middleware under `registerMiddleware()`, also rename `registerMiddleware()` to `registerMiddlewares()`